conda_lock/src_parser/__init__.py

```diff
@@ -101,6 +105,66 @@ def validate_hash(cls, v: HashModel, values: Dict[str, typing.Any]) -> HashModel
         return v
 
 
+class TimeMeta(StrictModel):
+    """Stores information about when the lockfile was generated."""
+
+    created_at: str = Field(..., description="Time stamp of lock-file creation time")
+
+    @classmethod
+    def create(cls) -> "TimeMeta":
+        import time
+
+        return cls(created_at=f"{time.asctime(time.gmtime(time.time()))}")
```

Comment:
Could you use an ISO 8601 formatted value here instead of `asctime` which is awkward to parse programmatically?